contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
+| ~SPC m d u~ or ~SPC m d D | Return from definition          |
```

Comment:
I agree with all this, for now let's go for `SPC m g b` as it is nice to type (at least on QWERTY).

We should also set `SPC m g ,` and `SPC m g .` but I think we may be able to do it later with a key translation that works for everywhere.